.changeset/tasty-pillows-think.md

```diff
@@ -0,0 +1,5 @@
+---
+'@example/next-app': patch
```

Comment:
Just btw, the examples don't need changesets

Reply:
Worth removing?

Reply:
Not really, doesn't matter